samples/metrics_monitor/test/igt_load.cpp

```diff
@@ -174,8 +174,18 @@ unsigned intel_gen(uint16_t devid)
 
 unsigned intel_gt(uint16_t devid)
 {
+    const struct intel_device_info *devinfo = intel_get_device_info(devid);
     unsigned mask = intel_gen(devid);
```

Comment:
move declaration to line 188 to be closer to usage

Reply:
Done